cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedTimeScheduleStore.java

```diff
@@ -493,7 +583,7 @@ private void upgradeJobs() throws Exception {
     }
   }
 
-  private void upgradeTriggers() throws Exception {
+  private void upgradeTriggers(Table table) {
```

Comment:
also need to check whether the trigger after upgrade already exists, don't add persist trigger if it already exists